lib/membrane_rtc_engine/engine.ex

```diff
@@ -525,19 +594,82 @@ defmodule Membrane.RTC.Engine do
     {{:ok, tracks_msgs}, state}
   end
 
-  @spec publish(msg :: any()) :: Keyword.t()
+  @doc """
+  This function create a notification, which should be send to all others endpoints in room.
+  """
```

Comment:
Maybe something like 
```suggestion
  @doc """
  Publishes message to all other endpoints in RTC Engine.
  
  This function returns `Membrane.Element.Action.notify_t()` that should be returned from Membrane callback.
  """
```
We should also fix typespec for these functions as they should return list of this type https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:notify_t/0

FYI the first sentence always appears in Functions summary section in hex docs so it is worthed to write it as simple as possible :wink: 

![image](https://user-images.githubusercontent.com/24651552/143481971-1a1ea929-0155-4bd6-a994-56ae02859b98.png)

Reply:
I am not sure if this proposed description is correct. Because this function will not publish this message to all other endpoints.  So maybe something like this?

```markdown
  Create notification, which is responsible for publishes message to all other endpoints in RTC Engine.

  This function returns `Membrane.Element.Action.notify_t()` that should be returned from Membrane callback.
```